scavenger-api/src/main/kotlin/com/navercorp/scavenger/service/SnapshotNodeService.kt

```diff
@@ -75,22 +75,23 @@ class SnapshotNodeService(
 
     private fun updateCountGraph(
         current: Node,
-        elements: List<String>,
+        elements: List<Map<String, String>>,
```

Comment:
How about writing a data class instead of a map?

Reply:
Good. I had to check the safety when I used the map.
If we use a data class, we don't have to check it

Reply:
fixed in b2d0a4aefe75de4a306302006bc9df595635eac3